src/app/marketing/page.tsx

```diff
@@ -826,6 +826,8 @@ export default async function Marketing() {
         </div>
       </div>
 
+      {/* <Projects /> */}
```

Comment:
Seems like you're including `/public/dummyImage.png`, `/src/app/marketing/components/Projects.jsx` in this PR accidentally - can you remove them?